sys-libs/libseccomp/libseccomp-2.5.4.ebuild

```diff
@@ -42,8 +43,19 @@ PATCHES=(
 )
 
 src_prepare() {
+	if use experimental-loong; then
+		PATCHES+=( "${WORKDIR}"/libseccomp-2.5.4-loongarch64-20220425.patch )
```

Comment:
use P here to be consistent with SRC_URI

Reply:
done